nsdb-http/src/main/scala/io/radicalbit/nsdb/web/CustomSerializers.scala

```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright 2018 Radicalbit S.r.l.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.radicalbit.nsdb.web
+
+import io.radicalbit.nsdb.common.statement._
+import org.json4s.JsonAST.{JDouble, JField, JInt, JLong}
+import org.json4s.{CustomSerializer, JNull, JObject, JString}
+
+object CustomSerializers {
+
+  val customSerializers = List(
+    AggregationSerializer,
+    ComparisonOperatorSerializer,
+    LogicalOperatorSerializer,
+    OrderOperatorSerializer,
+    LikeExpressionSerializer,
+    EqualityExpressionSerializer
+  )
+
+  val customSerializersForTesting = customSerializers ++ List(RelativeComparisonSerializerForTesting)
+
+  case object AggregationSerializer
+      extends CustomSerializer[Aggregation](_ =>
+        ({
+          case JString(aggregation) =>
+            aggregation match {
```

Comment:
here, in order to reinforce this check, I would all a lowercase in order not to check it in a case sensitive fashion